q2_diversity/_alpha/_visualizer.py

```diff
@@ -391,8 +392,9 @@ def alpha_rarefaction(output_dir: str, table: biom.Table, max_depth: int,
                 filenames.append(jsonp_filename)
 
         with open(os.path.join(output_dir, filename), 'w') as fh:
-            data.columns = ['depth-%d_iter-%d' % (t[0], t[1])
-                            for t in data.columns.values]
+            data.columns = [
+                '_alpha_rarefaction_depth_column_-%d_iter-%d' % (t[0], t[1])
```

Comment:
Do we need to change the name here? If this is just for the sake of writing out a TSV file with the results in it we might be able to keep the old name, which might be a bit more clear for users.